testing/test-runtime/build.rs

```diff
@@ -90,10 +90,6 @@ async fn run() {
         #[subxt::subxt(
             runtime_metadata_path = "{}",
             derive_for_all_types = "Eq, PartialEq",
-            substitute_type(
-                type = "sp_arithmetic::per_things::Perbill",
-                with = "::sp_runtime::Perbill"
```

Comment:
Yes exactly you can't do this anymore

Reply:
Yeah; you'd have to copy the type over or substitute it with some wrapper like `Static<::sp_runtime::PerBill>`